RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporter.cc

```diff
@@ -185,7 +191,7 @@ void GeneralTracksImporter::importToBlock(const edm::Event& e, BlockElementImpor
         TrackProdIDKey trk = std::make_pair(pftrackref->trackRef().id(), pftrackref->trackRef().key());
         auto lower = std::lower_bound(vetoed.begin(), vetoed.end(), trk);
         bool inVetoList = (lower != vetoed.end() && *lower == trk);
-        if (!inVetoList || (vetoMode_ == pfCandidateCollection && muonref.isNonnull())) {
+        if (!inVetoList || (vetoMode_ == pfRecTrackCollection && muonref.isNonnull())) {
```

Comment:
is this the reason for the change?

Reply:
the comment is based on the diffs visible in https://github.com/cms-sw/cmssw/pull/32291/files/0df27e6ad11fc7a4aa344afbff4a804c59d1d407..3e3bd552c8bfd74735fe0b949357afbbbb94477e
which apparently correspond to the new code since the previous working test and before my review was implemented

Reply:
> is this the reason for the change?

I think so. Sorry that I didn't respond more directly to your question earlier.